### PR TITLE
Changing optional to jargonExtra

### PR DESCRIPTION
**Pull request by alexvonme**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ To add a new rule, we suggest following these steps:
 8. If useful, create a folder named "original_files" and add relevant original files to that directory.
 9. Add the rule to the vale/styles/jargonLint or vale/styles/optional directory.
     - To jargonLint, if the rule is generally applicable.
-    - To optional, if the rule is reserved for specific situations.
+    - To jargonExtra, if the rule is reserved for specific situations.
```
Comment (ccoVeille) [inline CONTRIBUTING.md:41]:
```suggestion
    - To `jargonExtra` directory, if the rule is reserved for specific situations.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ To add a new rule, we suggest following these steps:
 8. If useful, create a folder named "original_files" and add relevant original files to that directory.
 9. Add the rule to the vale/styles/jargonLint or vale/styles/optional directory.
     - To jargonLint, if the rule is generally applicable.
```
Comment (ccoVeille) [inline CONTRIBUTING.md:40]:
```suggestion
     - To `jargonLint` directory, if the rule is generally applicable.
```

**Review by ccoVeille — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (alexvonme) [reply CONTRIBUTING.md:40]: The line above states directories so I don't think this is necessary.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -36,9 +36,9 @@ To add a new rule, we suggest following these steps:
     - The test file with accepted phrases should be "(RuleName).good.txt"
     - The test file with denied phrases should be "(RuleName).bad.txt"
 8. If useful, create a folder named "original_files" and add relevant original files to that directory.
-9. Add the rule to the vale/styles/jargonLint or vale/styles/optional directory.
+9. Add the rule to the vale/styles/jargonLint or vale/styles/jargonExtra directory.
```
Comment (alexvonme) [inline CONTRIBUTING.md:39]: I think this updates line should be enough to point that they are both directories

**Review by ccoVeille — APPROVED** [review]

**Pull request merged**